### PR TITLE
Add pipeline using Wrangler to generate lineage

### PR DESCRIPTION
**Pull request by yukiej**
Adding a pipeline to generate field level lineage for testing purposes.

### CODE REVIEW

**Comment by yukiej** [comment]
Build: https://builds.cask.co/browse/CDAP-UDUT397-1

File: cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                    "label": "Airport_source",
+                    "artifact": {
+                        "name": "core-plugins",
+                        "version": "2.3.0-SNAPSHOT",
+                        "scope": "SYSTEM"
+                    },
+                    "properties": {
+                        "copyHeader": "true",
+                        "schema": "{\"type\":\"record\",\"name\":\"etlSchemaBody\",\"fields\":[{\"name\":\"body\",\"type\":\"string\"}]}",
+                        "path": "file:/Users/yukijung/Desktop/Data/airports.csv",
```
Comment (itsanudeep) [inline cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json]: path should be   `"path": "file:/tmp/cdap-ui-integration-fixtures/airports.csv",` ,right?

File: cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    "artifact": {
+                        "name": "core-plugins",
+                        "version": "2.3.0-SNAPSHOT",
+                        "scope": "SYSTEM"
+                    },
+                    "properties": {
+                        "suffix": "yyyy-MM-dd-HH-mm",
+                        "format": "csv",
+                        "referenceName": "Airport_sink",
+                        "path": "/Users/yukijung/Desktop/Data/Sinks"
```
Comment (itsanudeep) [inline cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json]: path? `"path": "/tmp/cdap-ui-integration-fixtures",`

Comment (yukiej) [reply cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json]: Fixed!

File: cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json (hunk 1 above)
Comment (yukiej) [reply cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json]: Fixed!

**Comment by itsanudeep** [comment]
LGTM :+1:

**Review by itsanudeep — APPROVED** [review]

**Comment by yukiej** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-UDUT397-4

**Pull request merged**